docs/plugins-create.md

```diff
@@ -204,7 +204,7 @@ asdf plugin test <plugin-name> <plugin-url> [--asdf-tool-version <version>] [--a
 ```
 
 Only the two first arguments are required.
-If __version_ is specified, the tool will be installed with that specific version. Defaults to whatever returns `asdf latest <plugin-name>`.
+If \__version_ is specified, the tool will be installed with that specific version. Defaults to whatever returns `asdf latest <plugin-name>`.
```

Comment:
What is going on with the `\_` here?

Reply:
I making fixes in format in this file.
please review files
thanks for marking the errors